tests/repository_dependencies_shed.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<repositories description="A suite of Galaxy tools designed to work with version 1.2 of the SAMtools package.">
+    <repository changeset_revision="cf875cbe2df4" name="data_manager_sam_fasta_index_builder" owner="devteam" toolshed="https://toolshed.g2.bx.psu.edu" />
```

Comment:
maybe test some without hardcoded `changeset_revision`? Not strictly necessary, just being picky since I'm a bottle and a half in :P